### PR TITLE
About Page

### PR DESCRIPTION
**Pull request by dirkkelly**
https://deploy-preview-248--breadtubetv.netlify.com/about/

### CODE REVIEW

**Comment by samirillion** [comment]
This is great!

I can't honestly say I'd change anything without knowing more about the project; seems like you hit all the main points I was curious about pretty well. Thanks!

File: content/about/_index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+type: page
+---
+
+## Introduction
+
+[BreadTube.tv](/) is the collaborative effort of Creators, Subscribers, Moderators, Developers, Designers, and Coordinators working together to list content that is posted on [r/BreadTube](/reddit) and related mediums.
+
+BreadTube is a reference to the left-wing anarchist Peter Kropotkin's 1982 book, ["The Conquest of Bread"](https://theanarchistlibrary.org/library/petr-kropotkin-the-conquest-of-bread). The goal of the community is to challenge the far-right content creators who have taken power on services like YouTube for the purpose of spreading hate and to instead educate people on how their world operates, the alternative possible visions for our future, and how we organize ourselves to get there.
+
+> ["The Making of a YouTube Radical" New York Times](https://www.nytimes.com/interactive/2019/06/08/technology/youtube-radical.html)
```
Comment (jaymickey) [inline content/about/_index.md]: Excellent addition!

File: content/about/_index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+This website is a resource for discovering and sharing BreadTube content, content is suggested by public users through various forms ([Channels](/channels/new), [Playlists](/playlists/new)), these submissions are re viewable by interested members on [Discord](httsp://breadtube.tv/discord) and then added to the source code by a team of volunteer developers.
```
Comment (jaymickey) [inline content/about/_index.md]: > `are re viewable by interested members on`

reviewable? Think there is a cheeky space in there.

I would change this to: `can be reviewed by any interested community members on...and later added to the source code...`, it's a small change but I think it puts greater emphasis on the community.

File: content/about/_index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+Services such as YouTube use hidden algorithms to serve content to users, the preferences of these algorithms are determined by YouTube's values, which as they are a subsidiary of Google is  the generation of capital through Advertising.
```
Comment (jaymickey) [inline content/about/_index.md]:
```suggestion
Services such as YouTube use hidden algorithms to serve content to users, the preferences of these algorithms are determined by YouTube's values, which as they are a subsidiary of Google is the generation of capital through Advertising.
```

Another cheeky space.

File: content/about/_index.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
+---
+title: About
+type: page
+---
+
+## Introduction
+
+[BreadTube.tv](/) is the collaborative effort of Creators, Subscribers, Moderators, Developers, Designers, and Coordinators working together to list content that is posted on [r/BreadTube](/reddit) and related mediums.
+
+BreadTube is a reference to the left-wing anarchist Peter Kropotkin's 1982 book, ["The Conquest of Bread"](https://theanarchistlibrary.org/library/petr-kropotkin-the-conquest-of-bread). The goal of the community is to challenge the far-right content creators who have taken power on services like YouTube for the purpose of spreading hate and to instead educate people on how their world operates, the alternative possible visions for our future, and how we organize ourselves to get there.
```
Comment (jaymickey) [inline content/about/_index.md]:
```suggestion
BreadTube is a reference to the left-wing anarchist Peter Kropotkin's 1982 book - ["The Conquest of Bread"](https://theanarchistlibrary.org/library/petr-kropotkin-the-conquest-of-bread). The goal of the community is to challenge the far-right content creators who have taken advantage of the profit-driven algorithms used by services like YouTube for the purpose of spreading hate. Instead we wish educate people on how their world operates, the alternative possible visions for our future, and how we organize ourselves to get there.
```

**Comment by dirkkelly** [comment]
Thanks for the fixes @jaymickey!

**Pull request merged**